test/new-e2e/tests/gpu/gpu_test.go

```diff
@@ -97,6 +104,19 @@ type collectorStatus struct {
 	RunnerStats runnerStats `json:"runnerStats"`
 }
 
+func (v *gpuSuite) runCudaDockerWorkload() {
+	// Configure some defaults
+	vectorSize := 50000
+	numLoops := 100      // Loop extra times to ensure the kernel runs for a bit
+	waitTimeSeconds := 5 // Give enough time to our monitor to hook the probes
```

Comment:
i am ok with that called a sanity test, but we definitely should have "edge" cases that "challenge" our implementation as we will definitely face them in our customers' environments

Reply:
Yep, that's the idea, this is just the basic test (which hopefully shouldn't be flaky).